ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/mainSample.cpp

```diff
@@ -56,7 +58,7 @@
 #include "DrawOrderLayerListModel.h"
 
 #ifdef ESRI_BUILD
-#include "../../../../buildnum/buildnum.h"
+#include "../../../../../buildnum/buildnum.h"
```

Comment:
❓ Is this change related to this PR or to getting the builds switched over to the public repo? If the latter, better to keep it separate